buildSrc/src/main/groovy/com/uber/okbuck/core/model/android/AndroidAppTarget.groovy

```diff
@@ -113,27 +115,27 @@ class AndroidAppTarget extends AndroidLibTarget {
         if (minifyEnabled) {
             Set<File> proguardFiles = (baseVariant.mergedFlavor.proguardFiles +
                     baseVariant.buildType.proguardFiles)
-            if (proguardFiles.size() > 0 && proguardFiles[0].exists()) {
-                File genProguardConfig = getGenPath("proguard.pro")
+            Preconditions.checkArgument(proguardFiles.size() == 1, "%s proguard files found. Only one can be used.", proguardFiles.size())
```

Comment:
do you mean `proguardFiles.size() != 1`

Reply:
it throws an exception if the condition is not true. in this case we must have at least one proguard file since `minifyEnabled == true` but we cannot have more than one because okbuck will ignore it.